libnetwork/drivers/bridge/setup_ipv4_linux.go

```diff
@@ -28,6 +28,14 @@ func selectIPv4Address(addresses []netlink.Addr, selector *net.IPNet) (netlink.A
 }
 
 func setupBridgeIPv4(config *networkConfiguration, i *bridgeInterface) error {
+	// TODO(aker): the bridge driver panics if its bridgeIPv4 field isn't set. Once bridge subnet and bridge IP address
+	//             are decoupled, we should assign it only when it's really needed.
+	i.bridgeIPv4 = config.AddressIPv4
```

Comment:
I think the bridge driver should rely less on this field and only have a mandatory subnet. Bridge and gateway addresses should be optional. @corhere WDYT?